app/models/king.rb

```diff
@@ -1,2 +1,9 @@
 class King < Piece
```

Comment:
I think it's a good idea for everyone to write a spec for the piece type they are working on. Especially since we will implement more rules into #move_legal? in the future.